src/main/java/uk/gov/hmcts/reform/sandl.snlevents/controllers/SessionController.java

```diff
@@ -83,10 +86,12 @@ public ResponseEntity insertSession(@RequestBody UpsertSession upsertSession) th
 
     @PutMapping(path = "/update", consumes = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity updateSession(@RequestBody UpsertSession upsertSession) throws IOException {
-
-        UserTransaction ut = sessionService.updateSession(upsertSession);
-
-        return ok(ut);
+        try {
```

Comment:
could not that be handled in a global exception handler so we do not need to do try catch in each controller method that modifies the data